pkg/features/kube_features.go

```diff
@@ -599,6 +599,13 @@ const (
 	// Enables NetworkPolicy status subresource
 	NetworkPolicyStatus featuregate.Feature = "NetworkPolicyStatus"
 
+	// owner: @aravindhp @LorbusChris
+	// kep: http://kep.k8s.io/2271
+	// alpha: v1.26
+	//
+	// Enables viewing logs of node services using kubectl node-logs
+	NodeLogViewer featuregate.Feature = "NodeLogViewer"
```

Comment:
Nit: maybe this should be `NodeLogQuery` or similar? I think "Viewer" conveys a graphical interface or TUI but this is really an API to query / obtain node logs (which could then by used by some viewer).

Reply:
This was [suggested](https://github.com/kubernetes/kubernetes/pull/96120#discussion_r932758371) @ehashman. I am happy to make it `NodeLogQuery` given much has changed in this area.

Reply:
The "suggested" link in your comment seems to go to a seemingly comment from thockin, I also didn't think long on the proposed alternative name FWIW, but I think it's worth considering something a little clearer about the scope of the feature and NodeLogQuery at least seems closer, maybe 😅

Reply:
Sorry I have fixed the [link](https://github.com/kubernetes/kubernetes/pull/96120#discussion_r835617688).

Reply:
Done.